docs/guides/contributing.md

```diff
@@ -374,28 +374,25 @@ GitHub has additional documentation on [creating a pull request from a fork](htt
 
 [DoneJS.com](https://donejs.com/) is hosted on [GitHub pages](https://pages.github.com/) from the [`gh-pages`](https://github.com/donejs/donejs/tree/gh-pages) branch.
 
-First, from within your local DoneJS repository, create a new clone in the `site` folder and install the dependencies:
+First, from within your local DoneJS repository, make sure you have the latest changes and don’t have a local `gh-pages` branch:
 
 ```shell
-git clone git@github.com:donejs/donejs.git -b gh-pages site/
```

Comment:
I think that was true for the old site, but not for the new one. Now the process is:

1. Start off with an existing repo
2. [Build the docs](https://github.com/donejs/donejs/blob/d2a0522caf57eb4d56b86295f2ce45e18437a470/Makefile#L5)
3. [Switch to the site folder, make a commit, and push to `gh-pages`](https://github.com/donejs/donejs/blob/d2a0522caf57eb4d56b86295f2ce45e18437a470/Makefile#L6-L9)

I don’t think you need to make a new clone of just the `gh-pages` branch in the `site` folder anymore.